app/views/govuk_publishing_components/components/_cards.html.erb

```diff
@@ -0,0 +1,59 @@
+<%
+  heading ||= nil
+  items ||= nil
+  sub_heading_level = sub_heading_level || 3
+  two_column_layout ||= false
+  local_assigns ||= nil
+  shared_helper = GovukPublishingComponents::Presenters::SharedHelper.new(local_assigns)
+%>
+<% if items.present? %>
+  <div class="gem-c-cards">
+    <% if heading %>
+      <%= content_tag(shared_helper.get_heading_level, class: "gem-c-cards__heading govuk-heading-m") do %>
+        <%= heading %>
```

Comment:
I included it because there was an ask from some of our stakeholders to just use the grid on a particular page without including a heading above it. The heading wasn't seen as necessary, maybe because a grid of items can to a sighted user seem like a self-contained unit. But semantically it's a list so including a heading which explains what the list contains could help users, especially screen reader ones if they are not able to glance over the grid to get a sense of its contents.

So basically the heading is inside the component to encourage best practise so that teams always consider including a heading as part of the component as it's:
- a part of the component and the documentation
- used in all the examples

But I'd be interested in hearing if there's a different convention in this repo to solve this.

Reply:
I included it because there was an ask from some of our stakeholders to just use the grid on a particular page without including a heading above it. The heading wasn't seen as necessary, maybe (?) because a grid of items can seem like seem like a self-contained unit a sighted user. But semantically it's a list so including a heading which explains what the list contains could help users, especially screen reader users if they are not able to glance over the grid to get a sense of its contents.

So basically the heading is inside the component to encourage best practise so that teams always consider including a heading as part of the component as it's:
- a part of the component and the documentation
- used in all the examples

But I'd be interested in hearing what you think.

Reply:
I think your aim is laudable but the same thing could be achieved by writing this guidance in the documentation for this component. Having a heading inside this component adds unnecessary complexity and as far as I can tell it's exactly the same as putting a regular heading component before it.

Reply:
Considering these are navigational lists I find it reassuring to know that the component promotes the use of a landmark as a way of reaching out to this section. [contents list](https://components.publishing.service.gov.uk/component-guide/contents_list), and [summary list](https://components.publishing.service.gov.uk/component-guide/summary_list) are promoting the use of headings in a similar way.

Reply:
Since there is precedence for including a heading for a number of similar components and it promotes best practise and at least prompts teams to think about including a heading, I'm going to mark this  conversation as resolved. The component is still experimental and we can always revisit this if we get feedback to the contrary.